README.md

```diff
@@ -8,25 +8,16 @@ if (moreUsers) {
 }
 ```
 
-### Server Side Usage
 
-#### NConf-Based Config
```

Comment:
I'll fix this but we can still leave nconf instructions, just passing in the nconf config should work.